src/web/app/pages-student/student-profile-page/student-profile-page.component.ts

```diff
@@ -92,13 +98,8 @@ export class StudentProfilePageComponent implements OnInit {
     this.authService.getAuthUser().subscribe((auth: AuthInfo) => {
       if (auth.user) {
         this.id = auth.user.id;
-        const paramMap: { [key: string]: string } = {
-          user: this.user,
-          googleid: auth.user.id,
-        };
-
         // retrieve profile once we have the student's googleId
-        this.httpRequestService.get('/student/profile', paramMap).subscribe((response: StudentDetails) => {
+        this.studentProfileService.getStudentProfile(this.user, auth.user.id).subscribe((response: StudentDetails) => {
```

Comment:
You are not supposed to change GET method for student profile. It is not within the scope of the migration task.